### PR TITLE
Improve data loading

### PR DESCRIPTION
**Pull request by rennerocha**
- Data was outdated, so replaced the hard coded HTML with a simple scrape script that gets the latest version of the Wikipedia pages returning more accurate information of cities, towns and wards
- Instead of parsing the HTML file every time we import the library, create a script that generate automatically the dictionaries with the data needed to parse the addresses
- Applied black formatting to entire project

### CODE REVIEW

**Comment by rennerocha** [comment]
@kalessin 
I am using my version in a project, but given that this will have a more updated list of towns and wards, it would be great if you could release a new version (maybe a `0.2` version?) :smiley: 
If you are not using/maintaining this library anymore, I can handle the release process given the right permission.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
-     'ward_eng': 'Chūō'}
-
-    > parse('東京都江東区豊洲2丁目4-9')
-    {'prefecture': '東京都',
-     'prefecture_eng': 'Tokyo',
-     'unparsed_right': '4-9',
-     'ward': '江東区',
-     'ward_eng': 'Kōtō',
-     'city_district': '豊洲2丁目'}
+```bash
```
Comment (kalessin) [inline README.md:9]: shouldn't be python here?

**Review by kalessin — CHANGES_REQUESTED** [review]
The changes are ok to me. Thanks!

Only a minor fix in the README seems required, as I indicated.

**Review by kalessin — APPROVED** [review]
I will do that minimal change I asked.

**Comment by kalessin** [comment]
hi @rennerocha much thanks for the changes! they are excellent. I just released version 0.2 with your changes.

**Pull request merged**